tasks/modules/watcher.js

```diff
@@ -19,7 +19,7 @@ function createWatcher(watchPaths, callback) {
         });
     }
     function add(path, eventName, stats) {
-        if (_path.extname(path) !== ".ts") {
+        if (_path.extname(path) !== ".ts" || _path.extname(path) !== ".tsx") {
```

Comment:
This always returns true and causes the watcher functionality to break.
It should be &&.
Will submit a PR.